src/config/.htaccess

```diff
@@ -1,5 +1,27 @@
-deny from all
+# ------------------------------------------------------------------
+# Purpose of file: block any web access to unallowed files
+#                  stored under the /config directory
+# ------------------------------------------------------------------
+
+# Apache 2.2
+<IfModule !mod_authz_core.c>
+    Deny from all
```

Comment:
I see, you removed `Order deny,allow` everywhere so this is a consistent change. I assume `Order deny,allow` is the default.